perception/lidar_centerpoint/src/node.cpp

```diff
@@ -39,23 +39,23 @@ LidarCenterPointNode::LidarCenterPointNode(const rclcpp::NodeOptions & node_opti
 : Node("lidar_center_point", node_options), tf_buffer_(this->get_clock())
 {
   const float score_threshold =
-    static_cast<float>(this->declare_parameter<double>("score_threshold", 0.35));
+    static_cast<float>(this->declare_parameter<double>("score_threshold"));
```

Comment:
`score_threshold` has a different namespace in pointpainting. Sould not they be uniform?

Reply:
Fixed at https://github.com/autowarefoundation/autoware.universe/pull/6591/commits/f30631b30a14a03c426ce063e311c084db23a97a